lib/src/layout/scaffold.dart

```diff
@@ -1,117 +1,303 @@
+import 'dart:math' as math;
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
 import 'package:macos_ui/macos_ui.dart';
-import 'package:split_view/split_view.dart'; //todo: fork and migrate to nnbd
+import 'package:macos_ui/src/layout/content_area.dart';
```

Comment:
The `import`s below should also be `export`ed

Reply:
It's exported in `macos_ui.dart`